docs/database-structure.md

```diff
@@ -0,0 +1,336 @@
+# Schema Public
+
+## Tables
+
+### audit
+
+Some exercises require reviews from other students or any other member of the school. The reviewer checks that some conditions are met by the code of the student being reviewed. This process is called an `audit` and it's kept in this table.
+
+Columns:
+- id.
+- groupId: The group that the audit refers to.
+- auditorId: The user who's making the audit.
+- attrs: The attributes of the audits keep the feedback of the auditor for each question failed.
+- grade: 0 for failed and 1 for passed.
+- createdAt: The timestamp of the time and date of creation.
+- updatedAt: The timestamp of the last update.
+- code: The code that must be used by the auditor to start the audit.
+- resultId: The result of the audit.
+- version: A sha of the commit that the audit refers to.
+- endAt: The last date that the id refers to.
+- private: Used to access the code by the auditor.
+
+### discordToken
+
+Columns:
+- id
+- accessToken
+- refreshToken
+- expiresAt
+
+### event
+
+An event anchors an objects into time.
+> Note: Some [object attributes](attributes.md) are related to time, but they express time only in relative terms to the beginning of the event.
+> Also, live edits (i.e., editing an event while is ongoing) it's not supported. The changes will only take effect after reloading the page.
+
+Columns:
+- id
+- createdAt: The timestamp of the creation of the event.
+- endAt: The timestamp for the end of the event.
+- registrationId: The registration to the event.
+- objectId: The object that is used to generate the event.
+- parentId: The immediate event ancestor. The parent event.
+- status: Confirmation that the event finished correctly (NULL or done for now).
+- path: The path of the event (the relative url of the event).
+- campus: The campus that contains the event.
+- code: Used to unlock the exams.
+
+### event_user
+
+This table links users and events. A user can be in many events and an events have many users.
+
+Columns:
+- id
+- createdAt
+- userId
+- eventId
+- groupId (not used anymore, TODO: remove at some point)
+
+### group
+
+Some projects are required to be made by groups. Those groups are kept in this table.
```

Comment:
projects or raid maybe? (don't know if it's important to specify but as project is also something specific it could be confusing)